src/lib/Scenes/Favorites/index.tsx

```diff
@@ -60,10 +64,9 @@ class Favorites extends React.Component<Props, null> {
         <ScrollableTabView
           onChangeTab={selectedTab => this.fireTabSelectionAnalytics(selectedTab)}
           renderTabBar={props => (
-            <View>
-              <Title>Saves &amp; Follows</Title>
-              <ScrollableTabBar {...props} />
-            </View>
+            <TabBarContainer>
+              <TabBar {...props} />
+            </TabBarContainer>
```

Comment:
Ok, so this ended up being the major issue with the tab bar. Why using a scrollable tab bar is not performant isn’t clear yet, but we can work around that for now, as it doesn’t seem like we really need it to be scrollable. (This same scrollable tab bar is also used in Favourites and has the same type of bad performance, so we should switch it out there too.)

Aside from this issue, the issue is likely compounded by a few different CPU bound operations, which we'll try to reduce too.

Reply:
We added the Scrollable tab on the Favorites menu b/c there used to be a Saved Fairs options and we needed the scrolling to support the extra content. It can be removed for now of course, but I think the original reason for leaving it there was that we eventually intended to support saving fairs. (I think @l2succes  or @mbilokonsky can speak more to why we couldn't support saving fairs for the time being. But also not critical so this can wait) In the meantime I'll revert original tab in Favorites!

Reply:
@ashleyjelks Can you copy that message to https://artsyproduct.atlassian.net/browse/LD-476 ?

Reply:
I weighed in on the Jira ticket but will respond here too - as I understand it we scrapped the saved fairs feature because we found some subtle ambiguities in terms of what various fair references refer to. A little fuzzy, this was before my break, but if I remember correctly we have a lot of code that seems to treat `fair` and `fairOrganizer` interchangeably when they are very much not.

I think @anandaroop can speak more to this if you're curious, and I think Jun made the call to drop the feature.

Reply:
Thanks for the context 👌